ConsoleApp1/Program.cs

```diff
@@ -1,15 +1,118 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
-namespace ConsoleApp1
+namespace Lesson1
 {
     class Program
     {
         static void Main(string[] args)
         {
+            //Exercise 1
```

Comment:
As described in homework issue 4 exercises should be in 4 different projects in 1 solution, but you got 1 class, should be changed